client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx

```diff
@@ -80,16 +80,19 @@ export const SubscribersPageProvider = ( {
 	searchTermChanged,
 	sortTermChanged,
 }: SubscribersPageProviderProps ) => {
+	const { hasManySubscribers } = useManySubsSite( siteId );
 	const isDataView = isEnabled( 'subscribers-dataviews' );
 	const [ perPage, setPerPage ] = useState( DEFAULT_PER_PAGE );
 	const [ sortOrder, setSortOrder ] = useState< 'asc' | 'desc' >();
 	const [ dataViewSortTerm, setDataViewSortTerm ] = useState< SubscribersSortBy >(
 		SubscribersSortBy.DateSubscribed
 	);
+	const [ dataViewFilterOption, setDataViewFilterOption ] = useState< SubscribersFilterBy >(
+		hasManySubscribers ? SubscribersFilterBy.WPCOM : SubscribersFilterBy.All
```

Comment:
@arcangelini Does this initially filter out email subscribers for sites with many subscribers, and then add the filtering option to Dataviews?

Reply:
Fair point, I updated this.

Reply:
I just wasn't sure if we're keeping that logic. But maybe we need to test and decide?

Reply:
Oh I figured we would keep it for now and remove it after we test? Maybe find the upper limit.

Reply:
Sounds good!